spec/author_fields_spec.rb

```diff
@@ -0,0 +1,93 @@
+RSpec.describe SparqlToSwSolr::InstanceSolrDoc::AuthorFields do
+
+  let(:instance_uri) { 'http://ld4p-test.stanford.edu/1234567890#Instance' }
+  let(:isd) { SparqlToSwSolr::InstanceSolrDoc.new(instance_uri) }
+  let(:doc_hash) { isd.send(:add_author_fields, {}) }
+
+  context 'SPARQL' do
```

Comment:
I'm  not sure we need this context statement.  If we do (if tests won't pass without it), then it needs a better description than 'SPARQL'